gilded_rose.rb

```diff
@@ -1,48 +1,19 @@
+require 'cheese'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.